README.md

```diff
@@ -22,7 +26,16 @@ access to retrive ad information
   <summary>How to use the stack</summary>
 
 ### Using OCI terraform provider and Terraform CLI
+- Clone the repo.
+```bash
```

Comment:
Please include blank newlines around fenced code for consistency across Markdown rendering flavors.